### DISCUSSION TITLE
Any way to store backups in a local dir/nfs and not the cloud?

### DISCUSSION THREAD

**Discussion by YoNoSoyVictor** [cloudnative-pg/cloudnative-pg#2030] [Q&A]
I am considering this operator and I've been testing backup and recovery but realized the relevant section on the documentation only covers cloud solutions https://cloudnative-pg.io/documentation/1.20/backup_recovery/ . I could always do it by copying the pg_wal directory directly, but isn't there any operator-managed way for local or nfs backups?

**Comment by onedr0p** [comment]
You can selfhost a minio instance as described in the doc you linked. Or is there some other reason you must need local or NFS over selfhosting a tool like minio?

**Reply by YoNoSoyVictor** [reply]
Hi, thanks for taking the time to reply. It's just that my company already has an nfs server where most of the backups we can't put in the cloud are stored so I was trying to find the most convenient way to do it.

**Reply by onedr0p** [reply]
Personally I would deploy minio or (if using ceph) use cephs object gateway. Not having any form of s3 deployed onprem is only going to limit your options for this and other things.
